### PR TITLE
improved logging for public wandb project

### PR DESCRIPTION
**Pull request by afterpartyjohn**
cleaned up unused code in ValidatorLib and placeholder code in WandbLib as well.

### CODE REVIEW

File: conversationgenome/analytics/WandbLib.py [hunk 1]
```diff
@@ -26,53 +26,42 @@
...
+    def init_wandb(self, config=None, data=None):
+        if config:
+            #initialize data:
+            wallet = bt.wallet(config=config)
+            subtensor = bt.subtensor(config=config)
+            metagraph = subtensor.metagraph(config.netuid)
+            my_hotkey=wallet.hotkey.ss58_address
+            my_uid = metagraph.hotkeys.index(my_hotkey)
+        else: 
+            #initialize test data:
```
Comment (afterpartyai) [inline conversationgenome/analytics/WandbLib.py:38]: This should move before the IF and include all of the IF variables

File: conversationgenome/analytics/WandbLib.py [hunk 2]
```diff
@@ -26,53 +26,42 @@
...
 
-    def init_wandb(self, data=None):
+    def init_wandb(self, config=None, data=None):
+        if config:
+            #initialize data:
+            wallet = bt.wallet(config=config)
+            subtensor = bt.subtensor(config=config)
+            metagraph = subtensor.metagraph(config.netuid)
+            my_hotkey=wallet.hotkey.ss58_address
+            my_uid = metagraph.hotkeys.index(my_hotkey)
```
Comment (afterpartyai) [inline conversationgenome/analytics/WandbLib.py:36]: Some of these look fragile and this in particular was the same point of vulnerability we had to patch

File: conversationgenome/analytics/WandbLib.py [hunk 3]
```diff
@@ -26,53 +26,42 @@
...
         wandb_api_key = c.get("env", "WANDB_API_KEY")
         if not wandb_api_key:
             raise ValueError("Please log in to wandb using `wandb login` or set the WANDB_API_KEY environment variable.")
-        run = 5
+
         bt.logging.info("INIT WANDB", wandb_api_key)
 
-        my_uid = 2
-        PROJECT_NAME = 'cgp_test_run'
+        PROJECT_NAME = 'conversationgenome'
```
Comment (afterpartyai) [inline conversationgenome/analytics/WandbLib.py:51]: This should probably be a config with an IF statement for test runs

File: neurons/validator.py [hunk 4]
```diff
@@ -166,14 +167,16 @@ async def forward(self, test_mode=False):
...
                     (final_scores, rank_scores) = await el.evaluate(full_convo_metadata=full_conversation_metadata, miner_responses=responses)
 
+                    
+
                     for idx, score in enumerate(final_scores):
                         if self.verbose:
                             bt.logging.info(f"score {score}")
-                        uid = str(Utils.get(score, "uuid"))
+
+                        uid = str(self.metagraph.hotkeys.index(Utils.get(score, "hotkey")))
```
Comment (afterpartyai) [inline neurons/validator.py:176]: Fragile

File: conversationgenome/analytics/WandbLib.py (hunk 3 above)
Comment (afterpartyjohn) [reply conversationgenome/analytics/WandbLib.py:60]: For now, i added a netuid parameter to the logs for easy filtering

**Pull request merged**